app/models/form/setup/questions/scheme_id.rb

```diff
@@ -21,7 +21,7 @@ def answer_options
 
   def displayed_answer_options(case_log)
     organisation = case_log.owning_organisation || case_log.created_by&.organisation
-    schemes = organisation ? Scheme.select(:id).where(owning_organisation_id: organisation.id) : Scheme.select(:id)
+    schemes = organisation ? Scheme.select(:id).where(owning_organisation_id: organisation.id).where.not(managing_organisation_id: nil) : Scheme.select(:id).where.not(managing_organisation_id: nil)
```

Comment:
We have the `confirmed` field on `schemes` now. The idea was that that field is only set on the final "Create scheme" press, at which point all fields should have been validated. So we could check for that here rather than that each individual field has been set?

Doesn't make a huge difference now since it's currently just `managing_org_id` but when editing functionality is introduced and any field could potentially be missing it should make state management easier?

Reply:
Yeah, that's fair. I didn't realise we introduced that! Is that already setting the state?

Reply:
Yep, it should be being set when you confirm on the check answers page

https://github.com/communitiesuk/submit-social-housing-lettings-and-sales-data/blob/main/app/views/schemes/check_answers.html.erb#L134

Reply:
What about the data imports?

Reply:
Also, this doesn't seem to validate? I can create a record with values missing